backend/src/main/java/com/woowacourse/levellog/levellog/application/LevellogService.java

```diff
@@ -28,15 +28,16 @@ public class LevellogService {
     private final LevellogRepository levellogRepository;
     private final TeamRepository teamRepository;
     private final MemberRepository memberRepository;
+    private final TimeStandard timeStandard;
 
     @Transactional
     public Long save(final LevellogDto request, final Long authorId, final Long teamId) {
         final Team team = getTeam(teamId);
         final Member author = getMember(authorId);
         validateLevellogExistence(authorId, teamId);
 
-        final Levellog levellog = request.toLevellog(author, team);
-        final Levellog savedLevellog = levellogRepository.save(levellog);
+        final Levellog savedLevellog = levellogRepository.save(request.toLevellog(author, team));
```

Comment:
걍 levellog라고 하면 안되나요?
```suggestion
        final Levellog levellog = levellogRepository.save(request.toLevellog(author, team));
```